errbot/core.py

```diff
@@ -109,6 +109,8 @@ def _dispatch_to_plugins(self, method, *args, **kwargs):
             # noinspection PyBroadException
             try:
                 getattr(plugin, method)(*args, **kwargs)
+            except AttributeError:
```

Comment:
> In what scenario did you need to include this?

All of the default plugins (CommandError, Help, etc) would throw exceptions because their respective backends don't support reactions.  The current exception handler says that the plugin crashed, but that's not really the case, so I felt this was a better (and more accurate) description of the issue.

Reply:
Yeah, that makes sense.